pkgs/by-name/pa/panda3ds/package.nix

```diff
@@ -0,0 +1,199 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  makeWrapper,
+
+  xorg,
+  kdePackages,
+  cmake,
+  pkg-config,
+  imagemagick,
+
+  # (vulkan is currently not supported with qtgui and will cause a segfault, so don't even bother trying to compile it)
+  withVulkan ? stdenv.hostPlatform.isLinux && !withQtGui,
+  withOpenGL ? stdenv.hostPlatform.isLinux || withQtGui,
+  withMetal ? stdenv.hostPlatform.isDarwin,
+  withLuaJIT ? true,
+  withQtGui ? false,
+  withQtWayland ? false, # (broken)
```

Comment:
Is there any reason to have this option if it is broken? It could be re-added once support works with the package, and that wouldn't be a breaking change to re-add it.


Nit, non-blocking: Also, I'd personally prefer(though this is a personal thing) that the `withQtGUI` option be renamed to `withQt` instead, so that it can be a more generic option.

Reply:
yea ill just remove it

Reply:
Wayland now works on upstream now but X11 still seems to be less hit-or-miss for most people testing